src/styles/styles.js

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import styled from "styled-components";
+import "./styles.css";
+
+// Colors
+export const YELLOW = "#ECF955";
+export const GREEN = "#058855";
+export const PINK = "#E8A8CA";
+export const RED = "#DF4E20";
+export const ROYAL_BLUE = "#3C60DD";
+export const LIGHT_BLUE = "#13BCD7";
+export const BLACK = "#170E07";
+export const CREAM = "#FFFAF2";
+
+export const fonts = {
+  cubano: "Cubano",
+  epilogue: "Epilogue",
+  epilogueItalic: "Epilogue Italic",
+  IBMPlexMono: "IBM Plex Mono",
+};
+
+export const H1 = styled.h1`
+  font-family: ${fonts.cubano};
+  text-transform: lowercase;
+  font-size: 60px;
+  font-style: semi-bold;
+  color: {
+  }
+`;
+
+export const H2 = styled.h2`
+  font-family: ${fonts.epilogue};
+  font-size: 26px;
+  font-style: bold-italic;
+`;
```

Comment:
at the current moment H2 is ibm plex mono bold which I did not import; do you wanna add it to this PR? I can send you the ttf file

Reply:
also we have H3 now too! with a bold variant